src/DevTools.tsx

```diff
@@ -311,8 +311,6 @@ export default function DevTools<TCustomSettings>({
                   label="Close via outside click"
                   onChange={() => {
                     setCloseViaOutsideClick(!closeViaOutsideClick);
-                    // HACK: Forcing a reload because the setting isn't reflected until reload otherwise.
-                    window.location.reload();
```

Comment:
Setting wasn't being reflected because the outside click handler was closing over the original value of the state and not its up-to-date value.

Reply:
Good catch! Thanks!